### ISSUE TITLE
How to migrate

### ISSUE THREAD

**Issue by Delegator9176** [tux2000/home-assistant-addons#19]
Hi @tux2000 

many thanks for continuing the work of thebestmoshe. 
I'm using paperless ngx from thebestmoshe at the moment. As I have a lot of documents I would like to avoid to redo all the tagging etc. is (and if yes, can you show me?) there a way on how to easy migrate to your add-on without loosing all the data/work?

Thanks in advance

**Comment by tux2000** [comment]
Hej @nr458h,

indeed, paperless-ngx is a great tool and @TheBestMoshe did an amazing job starting this plugin for home assistant. Paperless-ngx comes with a set of management tools to, for example export all your settings and documents: https://docs.paperless-ngx.com/administration/#management-commands

I have used these tools before to migrate between versions. However, I'm sorry to say that i don't recall the exact process. I think for it to work with the home assistant plugins, I had to attach to that specific docker container and run the admin tools from within there...

Personally I started to use the [official docker-compose releases](https://docs.paperless-ngx.com/setup/#docker_hub) from within [portainer](https://github.com/MikeJMcGuire/hass-portainer) to run paperless-ngx along with home assistant (and then [ingress-proxy](https://github.com/tux2000/home-assistant-addons/tree/master/ingress-proxy) to expose the UI). I will try to add a more detailed description to the repo in the future...

Due to this, I will probably not work actively on updates to this plugin...
